### PR TITLE
Alphabetical sort of chem dispenser

### LINKED ISSUE #3000: Reagent dispenser entries should be alphabetised

### PR DESCRIPTION
**Pull request by Peptide90**
closes #3000

Did a sort on reagent dispenser to close out an issue. Should have been resolved by a previous PR.

### CODE REVIEW

**Comment by metalgearsloth** [comment]
TBH the UI should maybe auto-sort in case we ever get the chem names back

**Comment by Peptide90** [comment]
> 
> 
> TBH the UI should maybe auto-sort in case we ever get the chem names back

Yeah I couldn't find a way to do a sort within a YAML file so just thought I'd check through them. This commit should cover all the dispensers that have anything over like 2 items in.

Future PR's for dispensables should just be in alphabetical order anyway for good practice.

**Comment by metalgearsloth** [comment]
I just did it https://github.com/space-wizards/space-station-14/pull/3253

**Comment by metalgearsloth** [comment]
Also the reason is that if someone wants to just slap translated names onto it and still have the same reaction code they don't need to mess with the yaml.

File: Resources/Prototypes/Catalog/VendingMachines/seeds.yml [hunk 1]
```diff
@@ -4,16 +4,16 @@
   description: For when you need seeds fast. Hands down the best seed selection on the station!
   animationDuration: 1.3
   spriteName: seeds
-  startingInventory:
-    WheatSeeds: 10
+  startingInventory: #maintain in alphabetical order
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/VendingMachines/seeds.yml:7]: Seems silly to leave the comment on only the one so I'd just delete it.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Comment by Peptide90** [comment]
closing in favour of #3275 due to my github repo mess.

**Pull request closed without merging**